### PR TITLE
Bees

### PR DESCRIPTION
**Pull request by Brycey92**

### CODE REVIEW

File: main/modes/mode_bee.c [hunk 1]
```diff
@@ -0,0 +1,2016 @@
...
+    "Thinking bee!\n",
+    "Me?\n",
+    "Hold it. Let\'s just stop for a second. Hold it.\n",
+    "I\'m sorry. I\'m sorry, everyone. Can we stop here?\n",
+    "I\'m not making a major life decision during a production number!\n",
+    "All right. Take ten, everybody. Wrap it up, guys.\n",
+    "I had virtually no rehearsal for that.\n",
+};
+
+const song_t hereComesTheSun =
```
Comment (AEFeinstein) [inline main/modes/mode_bee.c:1608]: Nevermind, here's the song...

**Review by AEFeinstein — CHANGES_REQUESTED** [review]

File: main/modes/mode_main_menu.c [hunk 2]
```diff
@@ -230,8 +385,32 @@ void mainMenuButtonCb(buttonEvt_t* evt)
...
     mainMenu->autoLightDanceTimer = 0;
 
+    mainMenu->prevBtnState = mainMenu->btnState;
+    mainMenu->btnState = evt->state;
+
     if(evt->down)
     {
+        if(!mainMenu->debugMode)
+        {
+            if  (
```
[the remark this answers is not in the document]
Comment (Brycey92) [reply main/modes/mode_main_menu.c:392]: Oops, had that and I don't know where it went - fixed!

File: main/modes/mode_bee.c [hunk 3]
```diff
@@ -0,0 +1,2016 @@
...
+    .fnEspNowRecvCb = NULL,
+    .fnEspNowSendCb = NULL,
+    .fnAccelerometerCallback = NULL,
+    .fnAudioCallback = NULL,
+    .fnTemperatureCallback = NULL,
+    .overrideUsb = false
+};
+
+// Everyone's here
+static const char* creditNames[] =
```
[the remark this answers is not in the document]
Comment (Brycey92) [reply main/modes/mode_bee.c:62]: Fixed shadowed var, fixes for copyrighted content pending

File: main/modes/mode_bee.c (hunk 1 above)
Comment (Brycey92) [reply main/modes/mode_bee.c:1608]: fixes for copyrighted content pending

**Pull request closed without merging**